synapse/metrics/_exposition.py

```diff
@@ -41,7 +43,7 @@
 MINUS_INF = float("-inf")
 
 
-def floatToGoString(d):
+def floatToGoString(d: Any) -> str:
```

Comment:
It's `typeof(Sample.value)` and I had a really hard time figuring out what that was, since `prometheus_client` doesn't have typings.

Though it seems that `prometheus_client` pinky promises that it's an `int` or `float`:
https://github.com/prometheus/client_python/blob/master/prometheus_client/samples.py#L35-L39
I shall update the function!